packages/core/src/Cardano/util/computeImplicitCoin.ts

```diff
@@ -150,19 +155,24 @@ const getTxDeposits = (
  * On the other hand, the transaction summary/display could receive a transaction from a dApp,
  * and can have mixed certificates (foreign and ours), so we need the list of reward accounts and drepKeyHash
  * to be able to distinguish the deposits that are going to our rewardAccounts from the ones that could
- * potentially go to a different reward accounts that we dont control (same with reclaims).
+ * potentially go to a different reward accounts that we don't control (same with reclaims).
  */
 export const computeImplicitCoin = (
   { stakeKeyDeposit, poolDeposit }: Pick<Cardano.ProtocolParameters, 'stakeKeyDeposit' | 'poolDeposit'>,
-  { certificates, withdrawals }: Pick<HydratedTxBody, 'certificates' | 'withdrawals'>,
+  {
+    certificates,
+    proposalProcedures,
```

Comment:
Please add a unit test in `computeImplicitCoin.test.ts`.

Reply:
f65c3783fb should solve this